src/sass/modules/_m-schemaform.scss

```diff
@@ -195,6 +195,22 @@ label {
   }
 }
 
+legend.schemaform-label {
+  font-weight: inherit;
+  font-size: inherit;
+  color: inherit;
+  line-height: inherit;
+  max-width: 46rem;
+  padding-bottom: 0;
+}
+
+fieldset.schemaform-field-template {
+  margin-top: 3rem;
+  &.schemaform-first-field {
```

Comment:
Is this ampersand necessary?

Reply:
I believe so, since `schemaform-first-field` is added to the fieldset when it's the first field.